fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -731,6 +734,18 @@ public LogicalPlan visitRegularQuerySpecification(RegularQuerySpecificationConte
         });
     }
 
+    @Override
+    public LogicalPlan visitInlineTable(InlineTableContext ctx) {
+        List<UnboundOneRowRelation> exprsList = ctx.rowConstructor().stream()
+                .map(this::visitRowConstructor)
+                .collect(ImmutableList.toImmutableList());
+        LogicalPlan relation = exprsList.get(0);
+        for (int i = 1; i < exprsList.size(); i++) {
+            relation = new LogicalUnion(Qualifier.ALL, ImmutableList.of(relation, exprsList.get(i)));
+        }
```

Comment:
u could not construct a multi children union in parser. because the rules before merge set oepration only could process union with two children

Reply:
u could construct a AVL just like in visitSetOperation

Reply:
this is only two children, it's OK.